test/tests/problems/eigen_problem/initial_condition/tests

```diff
@@ -65,4 +65,15 @@
     expect_out = 'SNES Object:'
     requirement = "The system shall support extra power iterations and check eigenvalue."
   [../]
+
+  [./newton_with_exact_initialization]
+    type = 'RunApp'
+    input = 'ne_ic_no_free.i'
+    cli_args = 'Executioner/initial_eigenvalue=0.1999416740 Postprocessors/niter/type=NumNonlinearIterations'
+    issues = '#20454'
+    slepc_version = '>=3.13.0'
+    expect_out = 'Iteration 0 eigenvalue = 0.1999416740'
+    requirement = "The system shall support eigenvalue execution with an initial condition."
```

Comment:
Here I want to say initial condition because I basically giving an option for users to give ||Bx||_2 on initial to potentially skip a normalization. So I think saying an initial condition here is proper. The `initial_eigenvalue` is basically the targeted initial ||Bx||_2 for the initial x. What do you think?

Reply:
btw, we do not have this requirement yet in this tests file.

Reply:
I made this requirement more clearer.